src/structures/MessageMedia.js

```diff
@@ -43,6 +45,35 @@ class MessageMedia {
 
         return new MessageMedia(mimetype, b64data, filename);
     }
+
+    /**
+     * Creates a MessageMedia instance from a URL
+     * @param {string} url
+     * @param {number} [sizeLimit=0]
+     * @returns {Promise<MessageMedia>}
+     */
+    static fromUrl(url, sizeLimit = 0) {
+        return new Promise((resolve, reject) => {
+            const pUrl = new URL(url);
+            const mimetype = mime.getType(pUrl.pathname);
+
+            if (!mimetype) {
+                return reject(new Error('Unable to determine MIME type'));
```

Comment:
I think that rejecting when you are unable to get he mime type would be an issue, how about a param "ignoreMime" ?